src/org/mozilla/javascript/Parser.java

```diff
@@ -2288,13 +2288,21 @@ else if (symDeclType == Token.LP) {
     }
 
     private AstNode expr() throws IOException {
+        return expr(false);
```

Comment:
do we really need this extra step?

Reply:
you mean eliminate `private AstNode expr()` and refactor all callsites to call `private AstNode expr(boolean)` instead?

Reply:
yes, i guess eclipse can do this 4 you